Makefile

```diff
@@ -205,9 +205,16 @@ endurance-tests: export KOPIA_EXE ?= $(KOPIA_INTEGRATION_EXE)
 endurance-tests: build-integration-test-binary $(gotestsum)
 	 $(GO_TEST) $(TEST_FLAGS) -count=1 -parallel $(PARALLEL) -timeout 3600s github.com/kopia/kopia/tests/endurance_test
 
-robustness-tool-tests: $(gotestsum)
+robustness_tests: export KOPIA_EXE ?= $(KOPIA_INTEGRATION_EXE)
```

Comment:
typo: you have underscore instead of dash

Reply:
Thanks, good catch ✔️